src/components/CommentInfo/CommentInfo.scss

```diff
@@ -1 +1,12 @@
-//
+.CommentList {
```

Comment:
why do you have commentsList styles in commentsInfo file?

Reply:
My bad, removed it, Thanks!